charts/incubator/torrserver/values.yaml

```diff
@@ -0,0 +1,46 @@
+#
+# IMPORTANT NOTE
+#
+# This chart inherits from our common library chart. You can check the default values/options here:
+# https://github.com/k8s-at-home/library-charts/tree/main/charts/stable/common/values.yaml
+#
+
+image:
+  # -- image repository
+  repository: smailkoz/torrserver
+  # -- image tag
+  tag: 1.0.111
+  # -- image pull policy
+  pullPolicy: IfNotPresent
+
+# -- environment variables. See more environment variables in the [torrserver documentation](https://torrserver.org/docs).
+# @default -- See below
+env:
+  # -- Set the container timezone
+  TZ: UTC
+  # -- Set data dir
+  TS_CONF_PATH: /data
+  # -- Set torrent-tiles dir
+  TS_TORR_DIRL: /data/torrents
+# -- Allowing to use DLNA
+hostNetwork: true
```

Comment:
```suggestion

# -- Set this to `true` to allow using DLNA
hostNetwork: false
```

I don't mind having the option in the values.yaml, but I don't like setting it to `true` by default, especially since maybe not everybody will use DLNA